tests/sample/map_reuse.c

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// clang -O2 -Werror -c droppacket.c -o dropjit.o
+//
+// For bpf code: clang -target bpf -O2 -Werror -c droppacket.c -o droppacket.o
```

Comment:
Fix filenames in these comments... should be "map_reuse" not droppacket or dropjit